File: recaudacion/2.Codigo/frontend/src/app/core/models/unidadejecutora.ts
export class UnidadEjecutora {
    unidadEjecutoraId: number;
    nombre: string;
    secuencia: string;
    codigo: string;
    numeroRuc: string;
    direccion: string;
    correo: string;
    telefono: string;
    celular: string;
    estado: boolean;
    usuarioCreador: string;
    usuarioModificador: string;
    index:number;
}
export class UnidadEjecutoraFilter {
    pageNumber: number;
    pageSize: number;
    sortColumn:string;
    sortOrder: string;
    codigo:string;
    nombre:string;
    estado:boolean;
}

File: recaudacion/2.Codigo/backend/RecaudacionApiGuiaSalidaBien/Domain/UnidadEjecutora.cs
using System;

namespace RecaudacionApiGuiaSalidaBien.Domain
{
    public class UnidadEjecutora
    {
        public int UnidadEjecutoraId { get; set; }
        public string Secuencia { get; set; }
        public string Codigo { get; set; }
        public string Nombre { get; set; }
        public string NumeroRuc { get; set; }
        public string Direccion { get; set; }
        public string Correo { get; set; }
        public string Telefono { get; set; }
        public string Celular { get; set; }
        public bool Estado { get; set; }
    }
}
